must-gather/common

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+export KEPLER_NS="openshift-kepler-operator"
+export LOGFILE_NAME="gather-debug.log"
```

Comment:
To help with maintenance (and to make debugging easier) it is better to avoid side-effects as much as possible  when sourcing files. Ideally, it is better if the file only included variables (esp  globals) which is only used by the functions in that file.  Hence I would recommend moving `KEPLER_NS` to individual files. Yes we are repeating ourselves but I believe that is the lesser evil :) 

If you notice the use of `LOGFILE_NAME`, is unused in this file while a `LOGFILE_PATH` is required by the functions and that must be defined elsewhere.  I think it may be easier to encapsulate this by requiring the caller to invoke and `init_log` function that sets the `LOGFILE_PATH` and makes it readonly soon after.